app/nexus/internal/route/acl/policy/create_intercept.go

```diff
@@ -43,9 +43,10 @@ func guardPutPolicyRequest(
 		return apiErr.ErrUnauthorized
 	}
 
+	// Request "write" access to the ACL system for the SPIFFE ID.
 	allowed := state.CheckAccess(
-		spiffeid.String(), "*",
-		[]data.PolicyPermission{data.PermissionSuper},
+		spiffeid.String(), "spike/system/acl",
```

Comment:
These changes do not have any effect for the default mode of operation because if the request comes from SPIKE Pilot (with SPIKE Pilot SVID) then we assume that it can manage policies and secrets without requiring any policy attached to it.

The checks will only be useful if we want an arbitrary workload to create/read/list secrets or policies.